chromium_src/components/omnibox/browser/history_url_provider.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_OMNIBOX_BROWSER_HISTORY_URL_PROVIDER_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_OMNIBOX_BROWSER_HISTORY_URL_PROVIDER_H_
+
+#define DoAutocomplete                                                       \
+  DoAutocomplete(history::HistoryBackend* backend, history::URLDatabase* db, \
```

Comment:
please do it the other way to not depend on args list:
```
#define DoAutocomplete \
  NotUsed(); \
  friend class BraveHistoryURLProvider; \
  void DoAutocomplete
```
same for other files.

Reply:
:+1: makes sense, thank you!